### ISSUE TITLE
Not an issue, but a suggestion.

### ISSUE THREAD

**Issue by xDanub** [poggit-orphanage/FactionsPro#65]
/f overclaim is very powerful.

It would be really cool if you could add a dynamic to make it less powerful. What I had in mind is whenever a faction overclaims another faction it could remove that amount of strength from the over claiming faction.

For example if I was in Faction A with 500 strength and another player from Faction B with 1,000 strength overclaimed me, I would remain at 500 strength but they would lose the amount of strength I have and Faction B would now be at 500 strength. (since they lost 500 strength for over claiming me)

**Comment by Awzaw** [comment]
Good idea - not sure when I'd have time to add that though, so why not get coding and fork the repository?!

**Comment by xDanub** [comment]
I would love to, but I have no idea how to code.... I am just a server owner and would’ve loved to see this feature. 
Cheers xd

**Comment by Awzaw** [comment]
Please test the latest dev build on Poggit. I've added the option `OverClaimCostsPower` to the Prefs file, which will subtract the victims power from yours when set to `true`

**Comment by SOF3** [comment]
Please write a better issue title.

**Comment by xDanub** [comment]
@SOF3 Does that suit you?
